server/src/main/java/org/apache/druid/server/metrics/QueryCountStatsMonitor.java

```diff
@@ -46,7 +46,8 @@ public boolean doMonitor(ServiceEmitter emitter)
     final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder();
     Map<String, Long> diff = keyedDiff.to(
         "queryCountStats",
-        ImmutableMap.of("query/success/count", statsProvider.getSuccessfulQueryCount(),
+        ImmutableMap.of("query/total/count", statsProvider.getSuccessfulQueryCount() + statsProvider.getFailedQueryCount(),
```

Comment:
Should it include interrupted too? It looks like "failed" and "interrupted" are exclusive.

For consistency I'd also suggest retrieving the successful/failed/interrupted counts first, then summing them into the total. This is because the counts are always getting updated in the background, and if you call `getSuccessfulQueryCount()` more than once, the two won't necessarily be the same.

Reply:
I think metric names should add slashes when some different options are imaginable after the slash. With this, I think just `query/count` or `query/count/total` (with another option of `query/count/delta` is better